lib/c-api/src/wasm_c_api/types/extern_.rs

```diff
@@ -21,6 +21,16 @@ pub unsafe extern "C" fn wasm_extern_type(e: &wasm_extern_t) -> Box<wasm_externt
 #[no_mangle]
 pub unsafe extern "C" fn wasm_externtype_delete(_et: Option<Box<wasm_externtype_t>>) {}
 
+#[no_mangle]
+pub unsafe extern "C" fn wasm_externtype_copy(
```

Comment:
This function does not need to be `unsafe` as it's valid within a Rust context.

All `unsafe` means is that the Rust type system is insufficient to cover the behavior of the function. But when we use higher level types, this is often no longer the case.  And `unsafe` has no effect in regards to FFI.

This is a super minor change and it's a mistake I make constantly too and we have a lot of instances of this in the wasm C API, so I don't think it's worth blocking this PR over until we start doing a good job at removing it 😆

Reply:
Yeah, that was a reflex attached to this codebase :-D. Fixed.